src/www/ui/admin-dashboard-general.php

```diff
@@ -201,6 +213,15 @@ function DatabaseMetrics()
 
     $V .= "</table>\n";
 
+    $restRes[] = [
+      "metric" => $text,
+      "total" => intval($connection_count)
```

Comment:
Other metrics have `total` as string. This particular metric has it as an `int`. This causes the schema to fail. Please make sure this variable is also a string.

Reply:
Noted